docs/RegisteredFilterPlugins.md

```diff
@@ -1,18 +1,18 @@
-Registered Filter Plugins
--------------------------
-Please be sure to see HDF5 Filter Plugins, a convenience software that packages together many of the commonly used filters that users have created and registered.
+# Registered Filter Plugins
+
+Please see HDF5 Filter Plugins, a convenience software that packages together many commonly used filters users have created and registered.
```

Comment:
Is HDF5 Filter Plugins supposed to be a link? Or is this meant to be a definition?

Reply:
Shouldn't this be a link to the repo?